module/web/media/pyplex/css/styles.css

```diff
@@ -451,4 +457,20 @@ textarea.form-control:focus {
 .btn[disabled],
 fieldset[disabled] .btn {
   pointer-events: auto;
+}
+
+iframe#interactiveCaptchaIframeId
+{
+  position:relative;
+  width:465px;
```

Comment:
Do we need hardcoded values?

Reply:
Don't know, something that should be tested. 
I measured the size of the captcha on my screen, and limited the iframe to that size.

Don't know if the size of google captcha is the same on all devices. E.g. on high-resolution displays or tablets it may be different.

I agree that this should be changed, but  I don't know a quick solution of how to get the size of the captcha (before the checkbox to expand it).